R/unlockREDCap.R

```diff
@@ -278,9 +131,8 @@ connectAndCheck <- function(key, url, ...)
 #'          which returns the keys as a list. Use [globalenv()] to assign in the
 #'          global environment. Will accept a number such a '1' for global as well.
 #' @param keyring character. Potential keyring, not used by default.
+#' @param service character. The name to use in a yaml file for locating keys.
```

Comment:
Maybe refer back to the expected structure section where the service is "redcapAPI" (which is the default argument, and should rarely if ever be changed)?

Reply:
Side note, the section above the YAML structure (lines 94-98, at least in this version) reference "keyring_delete" and "backend_file". This needs to be cleaned up (use `shelter::keyring_delete`).